components/homme/cmake/HommeMacros.cmake

```diff
@@ -233,7 +233,8 @@ macro(createExecLib libName execType libSrcs inclDirs macroNP
   ADD_DEFINITIONS(-DHAVE_CONFIG_H)
 
   ADD_LIBRARY(${libName} ${libSrcs})
-  TARGET_INCLUDE_DIRECTORIES (${libName} PUBLIC ${inclDirs} ${modulesDir} ${CMAKE_CURRENT_BINARY_DIR})
+  TARGET_INCLUDE_DIRECTORIES (${libName} PUBLIC ${inclDirs} ${modulesDir} ${CMAKE_CURRENT_BINARY_DIR}
```

Comment:
Rather than add `PIO_INCLUDE_DIRS` to the include dirs, I think
```
target_link_libraries (${libName} piof)
```
should be more appropriate. Since targets transfer their props/deps, this would take care of fwd-ing all of PIO deps.

Reply:
Here again the issue is compiling, not linking. Note that EAMxx doesn't require Homme to do any linking.

Reply:
Works after the AD fix to the piof target.